patterns/structural/proxy.py

```diff
@@ -1,4 +1,11 @@
 """
+*What is this pattern about?
+Proxy is used in places where you want to add functionality to a class without changing its interface. The main class is called `Real Subject`. A client should use the proxy or the real subject without any code change, so both must have the same interface. Logging and controlling access to the real subject are some of the proxy pattern usages.
```

Comment:
Please split this long line. You don't have to make it strictly 80 characters long, but use an acceptable length for modern monitors (maybe 120 characters).

Reply:
Thanks a lot, @faif for your feedback!
This is my first open-source commitment.

I changed lengths of line to 80 char and changed the example to clarify more.